Dockerfiles/agent/secrets-helper/readsecret.py

```diff
@@ -0,0 +1,74 @@
+#!/opt/datadog-agent/embedded/bin/python
+
+import argparse
+import json
+import os.path
+import sys
+
+
+def list_secret_names(input_json):
+    query = json.loads(input_json)
+    if query["version"] != "1.0":
```

Comment:
Shouldn't we just check that the major is `1` ? We might add feature without breaking the format.